fedimintd/src/ui.rs

```diff
@@ -68,23 +68,9 @@ async fn run_page(axum::extract::State(state): axum::extract::State<MutableState
     RunTemplate {
         state: match state.dkg_state {
             Some(DkgState::Success) => {
-                let path = state
-                    .data_dir
-                    .join(CONSENSUS_CONFIG)
-                    .with_extension(JSON_EXT);
                 // TODO: refactor be a standalone function
```

Comment:
Is this TODO still relevant?

Reply:
Not sure on this TODO, but plan on removing this UI eventually.